reposcan/reposcan.py

```diff
@@ -48,13 +51,15 @@ def __init__(self, msg, success=True):
 
 class BaseHandler(RequestHandler):
     """Base handler setting CORS headers."""
+
     def data_received(self, chunk):
         pass
 
     def set_default_headers(self):
         self.set_header("Access-Control-Allow-Origin", "*")
         self.set_header("Access-Control-Allow-Headers", "Content-Type")
-        self.set_header("Access-Control-Allow-Methods", "GET, PUT, POST, DELETE")
+        self.set_header("Access-Control-Allow-Methods",
```

Comment:
please update the PR to not do such unnecessary line wraps

if we want to do a re-formatting, let's do it in separate task

Reply:
I am sorry about it, this had to be a job of `ctrl + shift + i`  formatting in vscode.